qemu/tests/balloon_boot_in_pause.py

```diff
@@ -16,14 +16,16 @@ class BallooningTestPause(BallooningTest):
     """
 
     def __init__(self, test, params, env):
-        super(BallooningTestPause, self).__init__(test, params, env)
-
+        self.test = test
+        self.params = params
+        self.env = env
         self.vm = env.get_vm(params["main_vm"])
         self.ori_mem = self.get_vm_mem(self.vm)
         if self.get_ballooned_memory() != self.ori_mem:
             self.balloon_memory(self.ori_mem)
         self.old_mmem = self.ori_mem
         self.old_gmem = None
+        self.sessions = {}
```

Comment:
I'm curious about why it could get LoginError due to the vm is being paused at this moment. IMO, this method should be invoked in the setup phase of the test, so the vm should not be paused at that time.

Reply:
The vm boots with "-S" option. So it is in paused status from the very beginning until we send "con" cmd.  The former L19 will call get_session()  and try to login before the vm is continued, so it would fail.

Reply:
OK, so we should fix it in the base class if we do need to run the test with `paused_after_start_vm = yes`, that because users could also get failures when running the balloon basic test with `paused_after_start_vm = yes`.

Reply:
@luckyh Thanks, updated. 
Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.balloon_boot_in_pause: PASS (190.28 s)